.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Artifacts for Spin Checklist app
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: 'new tag for this push'
+        required: true
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  echo-inputs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo Inputs
+        run: |
+          echo tag: ${{ github.event.inputs.tag }}
+
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 19
+
+      - name: setup spin
+        uses: fermyon/actions/spin/setup@v1
+        with:
+          version: v2.0.0
+          plugins: js2wasm
+          github_token: ${{ github.token }}
+
+      - name: npm install
+        run: npm install
+
+      - name: build and push
+        uses: fermyon/actions/spin/push@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          registry_username: ${{ github.actor }}
+          registry_password: ${{ secrets.GITHUB_TOKEN }}
+          registry_reference: "${{ env.REGISTRY }}/fermyon/spin-checklist:${{ github.event.inputs.tag }}"
```

Comment:
this is assuming we will move this repo to Fermyon org. if that is not the plan, we can change it to your username here.